content/en/docs/cluster-administration/persistent-volume-and-storage-class.md

```diff
@@ -156,6 +156,12 @@ NFS (Net File System) is widely used on Kubernetes with the external-provisioner
 
 ![Create Custom Storage Class](/images/storage/create-storage-class-storage-system-custom.png)
 
+{{< notice note >}}
+
+It is not recommended that you use NFS storage for production (especially on Kubernetes version 1.20 or later) as some issues may occur, such as `failed to obtain lock` and Pod `CrashLoopBackOff`. Besides, some apps may not be compatible with NFS, including [Prometheus](https://github.com/prometheus/prometheus/blob/03b354d4d9386e4b3bfbcd45da4bb58b182051a5/docs/storage.md#operational-aspects).
```

Comment:
A little comments regarding 
```
such as `failed to obtain lock` and Pod `CrashLoopBackOff`.
```
, I think they're not apple vs apple. Because the application `failed to obtain lock`, it then exited and restarted repeatedly (as  Pod `CrashLoopBackOff`).

How about 
```
such as `failed to obtain lock` and `input/output error`, which finally results in Pod `CrashLoopBackOff`
```

Reply:
Thank you for pointing it out. Changed.